aries_cloudagent/messaging/jsonld/tests/test_credential.py

```diff
@@ -329,6 +422,24 @@ async def test_sign_credential(input, mock_session):
     assert "jws" in result.get("proof", {}).keys()
 
 
+@pytest.mark.parametrize("input", TEST_SIGN_ERROR_OBJS)
+@pytest.mark.asyncio
+async def test_sign_dropped_attribute_exception(input, mock_session):
+    with pytest.raises(DroppedAttributeException):
+        await sign_credential(
+            mock_session, input.get("doc"), input.get("options"), TEST_VERKEY
```

Comment:
Probably better to use `input["doc"]` over `get`. `doc` being absent is a stronger error condition than defaulting to `None` and the `None` wouldn't cause an error immediately. "Fail fast, fix faster" :wink:

Reply:
Of course! How could I forget the Five F's of Fix-It Felix development. "Fail Fast, Fix Faster and don't Fail!" Remember the Four R's of review, "Review, Remember, Refocus and Review!"